Dockerfile

```diff
@@ -491,6 +491,11 @@ ADD --chown=buildbot package.json /opt/buildhome/test-env/package.json
 RUN cd /opt/buildhome/test-env && . ~/.nvm/nvm.sh && npm i --legacy-peer-deps &&\
     ln -s /opt/build-bin/run-build-functions.sh /opt/buildhome/test-env/run-build-functions.sh &&\
     ln -s /opt/build-bin/build /opt/buildhome/test-env/run-build.sh
+
 ADD --chown=buildbot tests /opt/buildhome/test-env/tests
 WORKDIR /opt/buildhome/test-env
-CMD . ~/.nvm/nvm.sh && npm test
+
+# Set `bats` as entrypoint
+ENTRYPOINT ["node_modules/.bin/bats"]
```

Comment:
What are your thoughts on using `["npx", "bats"]` instead?

Reply:
If I understand it correctly, `npx` would still force the sourcing of `nvm.sh`. This means effectively changing the environment we use to run our tests. I'm just concerned that by doing so we might be "hiding" some problem that our test env could spot?

Reply:
Ah, I see. Sounds good to me :+1:

Reply:
👍